src/_test_/getYoutubeData.test.js

```diff
@@ -1,47 +1,53 @@
+import axios from 'axios';
 import { mockData } from '../service/mockData';
 
-const useService = () => {
-  const getYoutubeData = () => {
-    return mockData.items;
+export const useService = () => {
+  const getYoutubeVideoSearch = (search, maxResults) => {
+    const URL =
+      'https://youtube.googleapis.com/youtube/v3/search?key=AIzaSyAqIkh7Mx1wSAncxU1N8BdZMXXwMsvAC3I&part=snippet&channelType=any&type=video';
+    const searchParams = `${URL}&q=${search}&maxResults=${maxResults}`;
+    return axios.get(searchParams).catch((error) => {
+      throw error;
+    });
```

Comment:
There is a function similar to this one defined in `src/utils/hooks/useService.js`, you can keep just one of them,